simcon_project/conversation_templates/views/template_management.py

```diff
@@ -0,0 +1,170 @@
+from django.views.generic import DeleteView
+from django.urls import reverse, reverse_lazy
+from django.shortcuts import render, redirect, get_object_or_404
+from conversation_templates.models import ConversationTemplate, TemplateFolder
+from conversation_templates.forms import FolderCreationForm
+from bootstrap_modal_forms.generic import BSModalCreateView, BSModalUpdateView, BSModalDeleteView
+from django_tables2 import TemplateColumn, tables, RequestConfig, A
+import re
+
+
+class FolderTemplateTable(tables.Table):
```

Comment:
Putting the django table class in the same file it's used seems like a cleaner method than what I did. It's unlikely anyone will need to use the table outside of the view it's intended for so it makes sense to put it in the same file. I think I'll copy you on this one :)

Reply:
Yeah I just put it in the same file since I don't think we are going to reuse these tables anywhere else. If we need to it's pretty simple to pull out and we also won't need to make specific file/folders just for tables too.

Reply:
Yep agreed. I think I'll need to make a little change to my PR or include it on another one.